test_pytest_timeout.py

```diff
@@ -603,3 +608,32 @@ def test_foo():
             "pytest_timeout_cancel_timer",
         ]
     )
+
+
+def test_session_timeout(pytester):
+    # 2 tests, each with 0.5 sec timeouts
+    # each using a fixture with 0.5 sec setup and 0.5 sec teardown
+    # So about 1.5 seconds per test, ~3 sec total,
+    # Therefore:
+    # A timeout of 1.25 sec should happen during the teardown of the first test
+    # and the second test should NOT be run
```

Comment:
i don't think this is asserted?  not immediately sure how you would assert it, maybe with more verbose output and more output lines to match.  or maybe there's something easier.

Reply:
i don't think this is addressed yet?